lib/pantry.rb

```diff
@@ -0,0 +1,29 @@
+class Pantry
+  attr_reader :stock
+  
+  def initialize
+    @stock = Hash.new(0)
+  end
+
+  def stock_check(ingredient)
+    @stock[ingredient]
+  end
+
+  def restock(ingredient, qty)
+    @stock[ingredient] += qty 
+  end
+
+  def enough_ingredients_for?(recipe)
+    enough = false 
+    @stock.each do |ingredient, qty|
+      recipe.ingredients_required.each do |item, quantity|
```

Comment:
I like the nested enumerable here, but the only problem is tht since this is going through every ingredient, the value of `enough` will be holding a true or false just based off of the last ingredient tested. Needed a .all? or something to test that there is enough in stock of every ingredient.